task-sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -54,6 +54,7 @@
 from pydantic import BaseModel, TypeAdapter
 
 from airflow.configuration import conf
+from airflow.sdk._shared.secrets_masker.secrets_masker import mask_secret
```

Comment:
We should definitely be consistent inside the project how we import this . Compare this to context.py just above

Reply:
Handled in 01ce9131f8